test_runner/performance/test_perf_pgbench_remote.py

```diff
@@ -0,0 +1,113 @@
+import dataclasses
+import os
+import subprocess
+from typing import List
+from fixtures.benchmark_fixture import PgBenchRunResult, ZenithBenchmarker
+import pytest
+from datetime import datetime
+import calendar
+import timeit
+import os
+import itertools
+
+pytest_plugins = ("fixtures.benchmark_fixture", )
+
+
+def utc_now_timestamp():
+    return calendar.timegm(datetime.utcnow().utctimetuple())
+
+
+@dataclasses.dataclass
+class PgBenchRunner:
```

Comment:
I see no false impression from that, this is just a convenient shortcut. Dataclasses are well supported by editors to give feedback based on type hints. I see no benefits in writing boilerplate init with all the type hints (which is also harder to read) instead of a dataclass.

Reply:
I'm with @LizardWizzard on this one. There's no indication that only POD structs can be dataclasses.

Reply:
¯\_(ツ)_/¯ then, I guess.